static/app/views/replays/detail/tagPanel/index.tsx

```diff
@@ -39,6 +41,10 @@ function TagPanel() {
   const organization = useOrganization();
   const {replay} = useReplayContext();
   const replayRecord = replay?.getReplay();
+  const tagFrames = replayRecord?.tags;
```

Comment:
nit:

I wouldn't call this `tagFrames` because the type is `Record<string, string[]>`. 
The`*Frame` (and `*Event`) suffix relates to specific types in https://github.com/getsentry/sentry/blob/master/static/app/utils/replays/types.tsx#L10-L14 and it might be confusing to see that name here again.

It'd just call it `const tags =`, but it's nbd either way.